contracts/oracles/BridgedUsdpOracle.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: bsl-1.1
+
+/*
+  Copyright 2022 Unit Protocol: Artem Zakharov (az@unit.xyz).
+*/
+pragma solidity 0.7.6;
+
+import "../interfaces/IOracleUsd.sol";
+import "../helpers/SafeMath.sol";
+import "../Auth.sol";
+
+/**
+ * @title BridgedUSDPOracle
+ * @dev Oracle to quote bridged from other chains USDP
+ **/
+contract BridgedUsdpOracle is IOracleUsd, Auth {
```

Comment:
`Auth2` may be used

Reply:
In `dev` (and some new networks) we don't have `Auth2` (and `Auth` has immutable `vaultParameters` there)